cmd/acra-connector/acra-connector.go

```diff
@@ -228,22 +269,50 @@ func main() {
 			Errorf("Configuration error: can't check is exists AcraConnector private key %v, got error - %v", clientPrivateKey, err)
 		os.Exit(1)
 	}
-	exists, err = utils.FileExists(serverPublicKey)
-	if !exists {
-		log.WithField(logging.FieldKeyEventCode, logging.EventCodeErrorWrongConfiguration).
-			Errorf("Configuration error: AcraServer public key %s doesn't exists", serverPublicKey)
-		os.Exit(1)
+	log.Infof("Client id and client key is OK")
+
+
+	// --------- check public key  -----------
+	if connectorMode == connector_mode.AcraTranslatorMode {
+		translatorPublicKey := fmt.Sprintf("%v%v%v_translator.pub", *keysDir, string(os.PathSeparator), *clientId)
+		exists, err = utils.FileExists(translatorPublicKey)
+		if !exists {
+			log.WithField(logging.FieldKeyEventCode, logging.EventCodeErrorWrongConfiguration).
+				Errorf("Configuration error: AcraTranslator public key %s doesn't exists", translatorPublicKey)
+			os.Exit(1)
+		}
+		if err != nil {
+			log.WithField(logging.FieldKeyEventCode, logging.EventCodeErrorWrongConfiguration).
+				Errorf("Configuration error: can't check is exists AcraTranslator public key %v, got error - %v", translatorPublicKey, err)
+			os.Exit(1)
+		}
+		log.Infof("AcraTranslator public key is OK")
 	}
-	if err != nil {
-		log.WithField(logging.FieldKeyEventCode, logging.EventCodeErrorWrongConfiguration).
-			Errorf("Configuration error: can't check is exists AcraServer public key %v, got error - %v", serverPublicKey, err)
-		os.Exit(1)
+
+	if connectorMode == connector_mode.AcraServerMode {
+		serverPublicKey := fmt.Sprintf("%v%v%v_server.pub", *keysDir, string(os.PathSeparator), *clientId)
```

Comment:
will be great to check key via keystore here too (or drop two checks by one with GetPeerPublicKey, because keystore now parameterized with mode)